pkg/universe.dagger.io/alpha/kubernetes/helm/upgrade.cue

```diff
@@ -125,11 +129,30 @@ import (
 					if dryRun {"--dry-run"},
 					if force {"--force"},
 					if cleanupOnFail {"--cleanup-on-fail"},
-					if username != _|_ {"--username=\(username)"},
-					if password != _|_ {"--password=${HELM_PASSWORD}"},
 				],
 				flags,
-			])
+			]),
+			" ")
+		env:  environment & {
```

Comment:
`"env"` to avoid name clashes. Look at this [page](https://docs.dagger.io/1241/field-shadowing) for complete explanation